modules/reset.tf

```diff
@@ -2,7 +2,18 @@
 resource "aws_sqs_queue" "account_reset" {
   name                       = "account-reset-${var.namespace}"
   tags                       = var.global_tags
-  visibility_timeout_seconds = 30
+  visibility_timeout_seconds = 60
+  redrive_policy = jsonencode({
```

Comment:
Create a queue redrive policy.  This will move messages that are causing issues into a dead letter queue if they are failing.